pennylane/circuit_graph.py

```diff
@@ -242,8 +199,7 @@ def observables_in_order(self):
         Returns:
             list[Observable]: observables
         """
-        nodes = [node for node in self._graph.nodes() if _is_observable(node)]
-        return sorted(nodes, key=_by_idx)
+        return self._observables
```

Comment:
the observables and operations are just intrinsically in the order that they occur in. So we can just return them.